runtime/integration-tests/src/generic/cases/liquidity_pools.rs

```diff
@@ -7452,7 +7452,7 @@ mod centrifuge {
 					pallet_balances::Pallet::<FudgeRelayRuntime<T>>::free_balance(
 						&Keyring::Alice.into()
 					),
-					109978937205
```

Comment:
This difference seems big tho.

Reply:
It makes sense because the relay has sent 3 and later received 1: 10 - 3 + 1 = 8 - fees

Reply:
Cool, thank you very much for checking, I forgot how these things were supposed to work :D

Reply:
Same result in rococo by the way